source/agent/addons/quic/binding.gyp

```diff
@@ -0,0 +1,64 @@
+{
+  'targets': [{
+    'target_name': 'quic',
+    'sources':[
+      'addon.cc',
+      "QuicFactory.cc",
+      'QuicTransportStream.cc',
+      'QuicTransportServer.cc',
+      'QuicTransportConnection.cc',
+      '../../../core/owt_base/MediaFramePipeline.cpp',
+      '../../../core/owt_base/MediaFrameMulticaster.cpp',
+    ],
+    'defines':[
+      'OWT_ENABLE_QUIC=1',
+    ],
+    'cflags_cc': [
+      '-std=gnu++14',
+      '-fno-exceptions',
+      '-DWEBRTC_POSIX',
+      '-DWEBRTC_LINUX',
+      '-DLINUX',
+      '-DNOLINUXIF',
+      '-DNO_REG_RPC=1',
+      '-DHAVE_VFPRINTF=1',
+      '-DRETSIGTYPE=void',
+      '-DNEW_STDIO',
+      '-DHAVE_STRDUP=1',
+      '-DHAVE_STRLCPY=1',
+      '-DHAVE_LIBM=1',
+      '-DHAVE_SYS_TIME_H=1',
+      '-DTIME_WITH_SYS_TIME_H=1',
+      '-Wno-non-pod-varargs',
+      '-fPIC',
+    ],
+    'include_dirs': [
+      "<!(node -e \"require('nan')\")",
+      'conn_handler',
+      '../../../core/common',
+      '../../../core/owt_base',
+      '../../../../build/libdeps/build/include',
+      # TODO: Fix it when CI for QUIC SDK is ready.
+      '/home/jianjunz/code/quic/src/owt/quic_transport/api',
```

Comment:
It's time for you to remove this hard code.

Reply:
Same as my previous comment: please remove the hard-code here with an environment variable + script warning when quic-agent build is enabled without this env set.

Reply:
These hard coded items will be removed once CI for QUIC SDK is finished.